doc/conf.py

```diff
@@ -321,7 +321,7 @@ def __getattr__(cls, name):
     'matplotlib', 'matplotlib.pylab', 'tifffile', 'EdfFile', 'netCDF4',
```

Comment:
I would rather keep them for a couple of versions as well because of the paper. 

As an alternative for the docs, is it also possible to remove the `.. automodule::` things from the relevant `tomopy.io` module docs and insert a link to http://dxchange.readthedocs.org/ instead? I am not sure whether this will remove the need to specify the other file format packages in this list, but maybe a small change in the importing code could make that possible as well.